<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="filterChat">
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="conversationsList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="messagesList">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="filterMember">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addMemberButton">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="userList">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="callButon">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
              </property>
              <property name="text">
               <string>Позвонить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="endButton">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
              </property>
              <property name="text">
               <string>Закончить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
              </property>
              <property name="text">
               <string>Выйти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="message">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled {
 background:#1A1A1A; 
color:#9B9B9B;
}</string>
          </property>
          <property name="text">
           <string>Отправить</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>chat.png</normaloff>chat.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Чат</string>
    </property>
    <addaction name="fullConversation"/>
    <addaction name="findPeople"/>
    <addaction name="newConversation"/>
    <addaction name="exit"/>
    <addaction name="separator"/>
    <addaction name="about"/>
    <addaction name="changePorts"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Аккаунт</string>
    </property>
    <addaction name="changeNick"/>
    <addaction name="disableSound"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="changeNick">
   <property name="text">
    <string>Изменить никнейм</string>
   </property>
  </action>
  <action name="disableSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выключить оповещения</string>
   </property>
  </action>
  <action name="newConversation">
   <property name="text">
    <string>Создать беседу</string>
   </property>
  </action>
  <action name="findPeople">
   <property name="text">
    <string>Найти людей</string>
   </property>
  </action>
  <action name="fullConversation">
   <property name="text">
    <string>Вся переписка</string>
   </property>
  </action>
  <action name="changePorts">
   <property name="text">
    <string>Изменить порты</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
